checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/api/session/SessionRequest.kt

```diff
@@ -4,10 +4,16 @@ import java.io.Serializable
 
 interface SessionRequest: Serializable
 
+/**
+ * [CardSessionRequest] is a serializable class that contains the request data for a VT session
```

Comment:
Worth mentioning that this **represents** the request body for the VT session endpoint

Reply:
updated